assets/tasks/common.json

```diff
@@ -1317,7 +1317,7 @@
         },
         {
             "label": "push-release-arm",
-            "detail": "These tasks will build the application based in the\nproduction Dockerfile and push the image to the DockerHub.",
+            "detail": "This task will build the application based in the\nproduction Dockerfile and push the image to the DockerHub.",
```

Comment:
This task will build the application based in the\nproduction Dockerfile and push the image to DockerHub.

Reply:
Just a small fix here before DockerHub

Reply:
done